build_tools/waf_package.py

```diff
@@ -41,51 +36,6 @@ def glob_files(patterns, cwd=None):
         os.chdir(oldcwd)
     return out
 
-# Original from  wafadmin/Build.py install_files:
-# I couldn't get the original code to work as I needed, so I made a small change to it.
-def package_install_files(self, path, files, env=None, chmod=Constants.O644, relative_trick=False, cwd=None):
```

Comment:
new waf doesn't have this function
it is possible to implement it using default waf functionality